pyvista/plotting/plotting.py

```diff
@@ -1639,9 +1639,7 @@ def add_volume(self, volume, scalars=None, clim=None, resolution=None,
             Scalars used to "color" the mesh.  Accepts a string name of an
             array that is present on the mesh or an array equal
             to the number of cells or the number of points in the
-            mesh.  Array should be sized as a single vector. If both
-            ``color`` and ``scalars`` are ``None``, then the active scalars are
```

Comment:
I think leaving this in is important as the user needs to know that active scalars are used if no scalars are entered in, but we should remove `color` as it's not a valid kwarg.

Also, could you fix "UnformGrid" on line 1635? Looks like that's a misspelling.